--- gtk-doc-1.18-pristine/Makefile.am	2011-06-17 10:39:37.000000000 +0200
+++ gtk-doc-1.18/Makefile.am	2012-06-02 11:42:18.774000015 +0200
@@ -40,7 +40,7 @@
 	up.png			\
 	style.css
 
-pkgconfigdir = $(datadir)/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = gtk-doc.pc
 
 aclocaldir = $(datadir)/aclocal
--- gtk-doc-1.18-pristine/Makefile.in	2011-09-14 15:52:31.000000000 +0200
+++ gtk-doc-1.18/Makefile.in	2012-06-02 11:42:05.015000017 +0200
@@ -323,7 +323,7 @@
 	up.png			\
 	style.css
 
-pkgconfigdir = $(datadir)/pkgconfig
+pkgconfigdir = $(libdir)/pkgconfig
 pkgconfig_DATA = gtk-doc.pc
 aclocaldir = $(datadir)/aclocal
 aclocal_DATA = gtk-doc.m4
