--- gsettings-desktop-schemas-3.6.1-pristine/schemas/org.gnome.desktop.background.gschema.xml.in.in	2013-02-23 16:05:43.217841601 +0100
+++ gsettings-desktop-schemas-3.6.1/schemas/org.gnome.desktop.background.gschema.xml.in.in	2013-02-23 16:15:13.320803105 +0100
@@ -15,7 +15,7 @@
       </_description>
     </key>
     <key name="picture-uri" type="s">
-      <default>'file://@datadir@/themes/Adwaita/backgrounds/adwaita-timed.xml'</default>
+      <default>'file://@datadir@/backgrounds/dropline-artwork/Dropline1.png'</default>
       <_summary>Picture URI</_summary>
       <_description>
         URI to use for the background image. Not that the backend only supports
