--- dropline-installer-2.12.1/src/dropline_applet.c-orig	2005-10-25 23:32:04.000000000 -0400
+++ dropline-installer-2.12.1/src/dropline_applet.c	2005-10-25 19:00:37.000000000 -0400
@@ -348,7 +348,7 @@
 	
 	curl_easy_setopt (session, CURLOPT_WRITEFUNCTION, net_write_cb);
 	curl_easy_setopt (session, CURLOPT_WRITEDATA, memory);
-	curl_easy_setopt (session, CURLOPT_URL, "http://www.dropline.net/gnome/DroplineFiles2c");
+	curl_easy_setopt (session, CURLOPT_URL, "http://diffie.droplinegnome.org/2.12.1/testing/x86_64/DroplineFiles2.12_x86_64");
 	curl_easy_setopt (session, CURLOPT_NOPROGRESS, TRUE);
 	curl_easy_setopt (session, CURLOPT_USERAGENT, "Dropline Alert " VERSION);
 
@@ -539,25 +539,40 @@
 
 static void
 applet_change_background_cb (PanelApplet *applet, 
-		PanelAppletBackgroundType type, GdkColor *color, const gchar *pixmap, 
+		PanelAppletBackgroundType type, 
+		GdkColor *color, 
+		//const gchar *pixmap,
+		GdkPixmap     *pixmap,
 		DroplineData *data)
 {
+	GtkRcStyle *rc_style;
+	GtkStyle *style;
+
 	gdk_threads_enter ();
 	
-	if (type == PANEL_NO_BACKGROUND) {
-		GtkRcStyle *rc_style = gtk_rc_style_new ();
-
-		gtk_widget_modify_style (data->applet, rc_style);
-		gtk_rc_style_unref (rc_style);
+	gtk_widget_set_style (GTK_WIDGET (data->applet), NULL);
+	rc_style = gtk_rc_style_new ();
+	gtk_widget_modify_style (GTK_WIDGET (data->applet), rc_style);
+	g_object_unref (rc_style);
+
+	switch (type) {
+		case PANEL_COLOR_BACKGROUND:
+			gtk_widget_modify_bg(GTK_WIDGET (data->applet), GTK_STATE_NORMAL, color);
+			break;
+
+		case PANEL_PIXMAP_BACKGROUND:
+			style = gtk_style_copy(GTK_WIDGET (data->applet)->style);
+			if (style->bg_pixmap[GTK_STATE_NORMAL]){
+				g_object_unref(style->bg_pixmap[GTK_STATE_NORMAL]);
+			}
+			style->bg_pixmap[GTK_STATE_NORMAL] = g_object_ref(pixmap);
+			gtk_widget_set_style(GTK_WIDGET (data->applet), style);
+			break;
+
+		case PANEL_NO_BACKGROUND:
+		default:
+			break;
 	}
-	else if (type == PANEL_COLOR_BACKGROUND) {
-		gtk_widget_modify_bg (data->applet,
-				      GTK_STATE_NORMAL,
-				      color);
-	} else { /* pixmap */
-		/* FIXME: Handle this when the panel support works again */
-	}
-	
 	gdk_threads_leave ();
 	
 	return;
@@ -744,52 +759,27 @@
 static void
 dropline_about_cb (BonoboUIComponent *uic, DroplineData *data, const gchar *verbname)
 {
-	static GtkWidget *about = NULL;
-	GdkPixbuf *pixbuf = NULL;
-	gchar *file;
 	static const gchar *authors [] = 
 	{
 		"Todd Kulesza <todd@dropline.net>",
+		"Francesco Gigli <jaramir@libero.it>",
+		"Michael Edgar <mgae@users.sourceforge.net>",
 		NULL
 	};
 	const gchar *documenters [] = 
 	{
 		NULL
 	};
-	const gchar *translator_credits = _("translator_credits");
-	
-	if (about)
-	{
-		gtk_window_set_screen (GTK_WINDOW (about), 
-				gtk_widget_get_screen (data->applet));
-		gtk_window_present (GTK_WINDOW (about));
-		
-		return;
-	}
-        
-	file = gnome_program_locate_file (NULL, GNOME_FILE_DOMAIN_PIXMAP, 
-			"dropline.svg", TRUE, NULL);
-	pixbuf = gdk_pixbuf_new_from_file (file, NULL);
-	g_free (file);
-
-	about = gnome_about_new (_("Dropline Update Alert"), VERSION, 
-			_("Copyright 2003-2004 Todd Kulesza"),
-			_("Receive alerts when new Dropline GNOME software is available."),
-			authors, documenters,
-			strcmp (translator_credits, "translator_credits") != 0 ? 
-					translator_credits : NULL,
-			pixbuf);
-
-	gtk_window_set_screen (GTK_WINDOW (about), 
-			gtk_widget_get_screen (data->applet));
-	gtk_window_set_wmclass (GTK_WINDOW(about), "dropline update alert", 
-			"Dropline Update Alert");
-	gnome_window_icon_set_from_file (GTK_WINDOW (about), 
-			GNOME_ICONDIR"/dropline.svg");
-	g_signal_connect (G_OBJECT (about), "destroy",
-		G_CALLBACK (gtk_widget_destroyed), &about);
-	
-	gtk_widget_show (about);
+    	gtk_show_about_dialog (NULL,
+		"name",			_("Dropline Update Alert"),
+		"version",		VERSION,
+		"copyright",		_("\xC2\xA9 2003-2004 Todd Kulesza\n\xC2\xA9 2006 Dropline GNOME Team"),
+		"comments",		_("Receive alerts when new Dropline GNOME software is available."),
+		"authors",		authors,
+		"documenters",		documenters,
+		"translator-credits",	_("translator-credits"),
+		"logo-icon-name",	"dropline",
+		NULL);
 	
 	return;
 }
@@ -866,11 +856,8 @@
 dropline_applet_factory (PanelApplet *applet, const gchar *iid, gpointer data)
 {
 	update_log (_("Creating applet...\n"));
-	
-	dropline_applet_create (applet);
-
+	dropline_applet_create(applet);
 	update_log (_("Applet created.\n"));
-	
 	return TRUE;
 }
 
