--- gnome-themes-standard-3.6.2-pristine/themes/Adwaita/gtk-2.0/gtkrc	2012-10-19 18:14:13.000000000 +0200
+++ gnome-themes-standard-3.6.2/themes/Adwaita/gtk-2.0/gtkrc	2013-02-22 15:50:11.991979299 +0100
@@ -986,7 +986,6 @@
 		file			= "Entry/combo-entry-border.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = LTR
 	}
 	image
 	{
@@ -997,7 +996,6 @@
 		file			= "Entry/combo-entry-border-disabled.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = LTR
 	}
         image
         {
@@ -1007,7 +1005,6 @@
 		file			= "Entry/combo-entry-border-active.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = LTR
         }
 
         # RTL version
@@ -1020,7 +1017,6 @@
 		file			= "Entry/combo-entry-border-rtl.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = RTL
 	}
 	image
 	{
@@ -1031,7 +1027,6 @@
 		file			= "Entry/combo-entry-border-disabled-rtl.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = RTL
 	}
         image
         {
@@ -1041,7 +1036,6 @@
 		file			= "Entry/combo-entry-border-active-rtl.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = RTL
         }
     }
 }
@@ -1061,7 +1055,6 @@
 		file			= "Entry/combo-entry-button.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = LTR
 	}
 	image
 	{
@@ -1070,7 +1063,6 @@
 		file			= "Entry/combo-entry-button.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = LTR
 	}
 	image
 	{
@@ -1079,7 +1071,6 @@
 		file			= "Entry/combo-entry-button-disabled.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = LTR
 	}
         image
         {
@@ -1088,7 +1079,6 @@
 		file			= "Entry/combo-entry-button-active.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = LTR
         }
 
         # RTL version
@@ -1099,7 +1089,6 @@
 		file			= "Entry/combo-entry-button-rtl.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = RTL
 	}
 	image
 	{
@@ -1108,7 +1097,6 @@
 		file			= "Entry/combo-entry-button-rtl.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = RTL
 	}
 	image
 	{
@@ -1117,7 +1105,6 @@
 		file			= "Entry/combo-entry-button-disabled-rtl.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = RTL
 	}
         image
         {
@@ -1126,7 +1113,6 @@
 		file			= "Entry/combo-entry-button-active-rtl.png"
 		border			= { 4, 4, 5, 4 }
 		stretch			= TRUE
-                direction               = RTL
         }
 
         }
@@ -1157,7 +1143,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-up-small.png"
       overlay_stretch		= FALSE
-      direction                 = LTR
     }
     image
     {
@@ -1169,7 +1154,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-up-small-prelight.png"
       overlay_stretch		= FALSE
-      direction                 = LTR
     }
     image
     {
@@ -1181,7 +1165,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-up-small-insens.png"
       overlay_stretch		= FALSE
-      direction                 = LTR
     }
     image
     {
@@ -1193,7 +1176,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-up-small-prelight.png"
       overlay_stretch		= FALSE
-      direction                 = LTR
     }
 
     # Spin-Up RTL
@@ -1207,7 +1189,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-up-small.png"
       overlay_stretch		= FALSE
-      direction                 = RTL
     }
     image
     {
@@ -1219,7 +1200,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-up-small-prelight.png"
       overlay_stretch		= FALSE
-      direction                 = RTL
     }
     image
     {
@@ -1231,7 +1211,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-up-small-insens.png"
       overlay_stretch		= FALSE
-      direction                 = RTL
     }
     image
     {
@@ -1243,7 +1222,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-up-small-prelight.png"
       overlay_stretch		= FALSE
-      direction                 = RTL
     }
 
     # Spin-Down LTR
@@ -1257,7 +1235,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-down-small.png"
       overlay_stretch		= FALSE
-      direction                 = LTR
     }
     image
     {
@@ -1269,7 +1246,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-down-small-prelight.png"
       overlay_stretch		= FALSE
-      direction                 = LTR
     }
     image
     {
@@ -1281,7 +1257,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-down-small-insens.png"
       overlay_stretch		= FALSE
-      direction                 = LTR
     }
     image
     {
@@ -1293,7 +1268,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-down-small-prelight.png"
       overlay_stretch		= FALSE
-      direction                 = LTR
     }
 
     # Spin-Down RTL
@@ -1307,7 +1281,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-down-small.png"
       overlay_stretch		= FALSE
-      direction                 = RTL
     }
     image
     {
@@ -1319,7 +1292,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-down-small-prelight.png"
       overlay_stretch		= FALSE
-      direction                 = RTL
     }
     image
     {
@@ -1331,7 +1303,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-down-small-insens.png"
       overlay_stretch		= FALSE
-      direction                 = RTL
     }
     image
     {
@@ -1343,7 +1314,6 @@
       stretch			= TRUE
       overlay_file		= "Arrows/arrow-down-small-prelight.png"
       overlay_stretch		= FALSE
-      direction                 = RTL
     }
   }
 }
